includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -770,8 +770,16 @@ public static function orders( $data = null ) {
 			) );
 		}
 
-		// By default, this is what is stored in payment meta.
-		$core_meta_keys = array(
+		/**
+		 * By default, this is what is stored in payment meta. These array keys are part of the core payment meta in 2.x
+		 * but are not needed as part of the order meta.
+		 * Extensions can add their keys to this filter if they use the payment meta array to store data and have
+		 * established a migration process to keep the data intact with the new order tables.
+		 *
+		 * @since 3.0
+		 * @param array The array of payment meta keys.
+		 */
+		$core_meta_keys = apply_filters( 'edd_30_core_payment_meta', array(
```

Comment:
I think you may be missing the closing `)` from `apply_filters()`.

Reply:
Ack! You are completely correct! Thanks for catching that!